src/coreclr/tools/aot/ILCompiler.RyuJit/Compiler/RyuJitCompilation.cs

```diff
@@ -23,14 +23,9 @@ public class RyuJitCompilation : Compilation
         private readonly ConditionalWeakTable<Thread, CorInfoImpl> _corinfos = new ConditionalWeakTable<Thread, CorInfoImpl>();
         internal readonly RyuJitCompilationOptions _compilationOptions;
         private readonly ProfileDataManager _profileDataManager;
-<<<<<<< HEAD
         protected readonly MethodImportationErrorProvider _methodImportationErrorProvider;
-        protected readonly int _parallelism;
-=======
-        private readonly MethodImportationErrorProvider _methodImportationErrorProvider;
         private readonly ReadOnlyFieldPolicy _readOnlyFieldPolicy;
-        private readonly int _parallelism;
->>>>>>> origin/runtime-main
+        protected int _parallelism;
```

Comment:
I suspect you can restore the `readonly` now:
```suggestion
        protected readonly int _parallelism;
```